tests/php/tests/includes/test_class.wp-job-manager-usage-tracking-data.php

```diff
@@ -53,6 +53,20 @@ class WP_Test_WP_Job_Manager_Usage_Tracking_Data extends WPJM_BaseTest {
 	 */
 	private $publish;
 
+	/**
+	 * Set up for tests.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		// Ensure job categories and types are enabled.
+		update_option( 'job_manager_enable_categories', 1 );
+		update_option( 'job_manager_enable_types', 1 );
```

Comment:
I'd rather enable these options explicitly in the individual test cases, as I also have tests that run when they're not enabled. The majority of test cases shouldn't need this set.